SmartPlaces.Facilities/samples/Topology/src/Properties/launchSettings.json

```diff
@@ -5,7 +5,7 @@
       "environmentVariables": {
         "AppInsightsConnectionString": "",
         "KeyVaultEndpoint": "",
-        "OntologyMappingFilename": "mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json",
+        "OntologyMappingFilename": "",
```

Comment:
I'm not sure this file should be checked in.. the `./SmartPlaces.Facilities/samples/Topology/src/appsettings.json` already defines this as the default. This local dev only override might lead to unanticipated differences in deployed code? Thoughts? Not strongly opinionated on this at this time.